CONTRIBUTING.md

```diff
@@ -0,0 +1,8 @@
+# Welcome!
+
+These are the ways you can currently contribute to the OSI Samples:
+
+- Give us [Feedback](https://feedback.osisoft.com/forums/922279-osisoft-github) at https://Feedback.osisoft.com/forums/922279-osisoft-github.  Please select the appropriate category when posting feedback.  This feedback can include new features, issues, requests for new samples, etc...
```

Comment:
Weird that the URL is capitalized here. "https://Feedback.osisoft.com" It just feels wrong. Feel free to ignore though.